src/Nutgram.php

```diff
@@ -287,4 +288,19 @@ public function resolve($callable)
 
         return $callable;
     }
+
+    /**
+     * Set my commands call to Telegram using all the registered commands
+     */
+    public function registerMyCommands(?array $opt = []): bool|null
+    {
+        $commands = [];
+        array_walk_recursive($this->handlers, static function ($leaf) use (&$commands) {
+            if ($leaf instanceof Command) {
```

Comment:
To avoid the "_Bad Request: command description must be non-empty_" error (when you don't wanna set a description to commands), please change to this:
```php
if ($leaf instanceof Command && !empty($leaf->getDescription())) {
   $commands[] = $leaf->toBotCommand();
}
```

Reply:
The goal is to make it work with a default also without being forced to set a description. Since '' is not supported, added a default string.